SRRecorderControl.m

```diff
@@ -149,7 +149,7 @@ - (instancetype)initWithFrame:(NSRect)aFrameRect
                                            forOrientation:NSLayoutConstraintOrientationVertical];
         }
 
-       NSString* sgwaesf = NSLocalizedStringFromTableInBundle(@"Click to record shortcut",
+       NSString* sgwaesf = NSLocalizedStringFromTableInBundle(@"Add Shortcut",
```

Comment:
the string in the macro should match whatever we want to show in the UI. so if we want "+ Shortcut" to be the official string, that's what these should all be

Reply:
tl;dr Agree with jballance, and don't forget the other languages.  More details below.

Normally in our own code, yes.  We usually just use the actual text for the key.  However, I am pretty sure we (TechSmith) changed it to "Add Shortcut" because the "Click to record shortcut" was too long.  But the way we changed it was to have the least amount of impact so we kept "Click to record shortcut" as the key and just changed the string to "Add Shortcut".  I feel there are 2 options:

1. Make the change with very little code impact.  This would involve changing 
`"Click to record shortcut" = "Add Shortcut";` in `ShortcutRecorder.strings` to 
`"Click to record shortcut" = "+ Shortcut";`

NOTE: This does cause us to send the wrong text to Rubric since we are sending the the key "Click to record shortcut" instead of the actual text we want.  I think we were OK with this at one time because we never really change this library... and might even have it blacklisted to not send to Rubric on Snagit (i would have to check that if we really care)

2. Update it as we have for all the strings in our other projects.  This would be what @jballance is suggesting.  Just make it
`NSLocalizedStringFromTableInBundle(@"+ Shortcut"`.  You will have to update it in all the locations you changed it to "Add Shortcut" but then it will at least go to Rubric correctly.

At first I was thinking 1 would be better as not to change as much stuff... but reading your note that "the longer wrong string is going to Rubric" if we did 2 if would fix that problem and work like the rest of the process.

OTHER THINGS TO NOTE:
Don't forget to update this in all the other `.strings` files for all the other languages.  I know we might not know the text for it yet... but since we are changing the key potentially we will want to change them.

FUTURE TECH DEBT:
On the main repo from this, we are SO far behind.  Some day we should update it and would probably get rid of some bugs.  Also, their strings files are formatted correctly with comments in the main repo :)

Reply:
@aprilnoren if we go with #2 and rubric gets the correct new string, are you okay with not updating the .strings in this PR, and just waiting for them to get corrected after our next drop with Rubric?

Reply:
The only reason I was using "Add Shortcut" as the reference string was because of some unnameable unjustifiable squeamishness around using symbols in the reference. But if y'all are ok with "+ Shortcut" on both sides of the equation, then I'll change that.

And agreed, @aprilnoren , we'll need to make sure these are updated in our respective consumer projects. In fact, CMac is behind enough that someone (I volunteer) will need to make sure all historical updates are being correctly reflected, or at least not breaking things.

Reply:
Well since these string files are a little different (not formatted correctly, meaning no comment lines and scrunched together...) I am not sure how well they "auto-update" from Rubric.  It is fine to not have the localized part correct... since we need Rubric to tell us what that is.  But I would recommend updating the 

`"Click to record shortcut" = "Add Shortcut";`
to
`"+ Shortcut" = "+ Shortcut"`
in all other language `.strings` files because

1. It will be easier to see what needs to be replaced if we have to update them manually when we get them from Rubric.
2. If you don't update it now we might accidentally leave the "Click to record shortcut" line in there which is dead/unneeded code
3. If it does auto update from Rubric, it won't be able to match the old line and will leave dead/unneeded code.